publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/persistence/ExpandedDataEntriesPersistenceHandler.java

```diff
@@ -22,42 +27,49 @@ public class ExpandedDataEntriesPersistenceHandler
     extends DestinationsEventBridgeEventHandler<EventReference, EventReference> {
 
     public static final String EXPANDED_ENTRY_PERSISTED_EVENT_TOPIC = "PublicationService.ExpandedEntry.Persisted";
-    public static final EventReference EMPTY_EVENT = new EventReference("Empty event", null);
+    public static final String SENT_TO_RECOVERY_QUEUE_MESSAGE = "DateEntry has been sent to recovery queue: {}";
     private static final Logger logger = LoggerFactory.getLogger(ExpandedDataEntriesPersistenceHandler.class);
+    private final QueueClient queueClient;
     private final S3Driver s3Reader;
     private final S3Driver s3Writer;
 
     @JacocoGenerated
     public ExpandedDataEntriesPersistenceHandler() {
-        this(new S3Driver(PublicationEventsConfig.EVENTS_BUCKET), new S3Driver(PERSISTED_ENTRIES_BUCKET));
+        this(new S3Driver(PublicationEventsConfig.EVENTS_BUCKET), new S3Driver(PERSISTED_ENTRIES_BUCKET),
```

Comment:
Btw this recovery queue has 14 days before it expires.
The objects in the event bucket expires after 1 day. :s

Reply:
Increase the lifecycle for objects in the events bucket please.

Reply:
Do we need it?
Now we persist recovery entry when we are sending event to eventbridge, not when reading incoming event from eventbridge.